src/components/container/MyDataHelpsAssistant/MyDataHelpsAssistant.css

```diff
@@ -0,0 +1,190 @@
+    --touch: 44px;
+    --lg-padding: 20px;
+}
+
+.mdh-assistant-collapsed {
+    position: fixed;
+    bottom: 20px;
+    right: 20px;
+    width: var(--touch);
+    height: var(--touch);
+    background: var(--mdhui-color-primary);
+    display: flex;
+    box-shadow: var(--mdhui-box-shadow-1);
+    border-radius: 200px;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    cursor: pointer;
+}
+
+.mdh-assistant {
+    position: fixed;
+    bottom: 0;
+    background-color: white;
+    font-size: 14px;
+    width: calc(100% - 1rem);
+    border: var(--border);
+    border-radius: 12px;
+    overflow: hidden;
+    margin: .5rem;
+    z-index: 3;
+    max-width: 800px;
+    right: 0;
+    box-shadow: var(--mdhui-box-shadow-1);
+}
+
+.mdh-assistant-header {
+    display: flex;
+    align-items: center;
+    height: 44px;
+    padding: 0 12px;
+    justify-content: space-between;
+    border-bottom: var(--border);
+    font-weight: 600;
+    color: var(--ce-grey-55);
+}
+
+.mdh-assistant-header .collapse-button {
+    color: var(--mdhui-color-primary);
+    cursor: pointer;
+}
+
+.mdh-assistant-header svg {
+    padding-right: 0.5rem
+}
+
+.mdh-assistant #log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    height: 80vh;
+}
+
+/* typography */
+.mdh-assistant p,
+.mdh-assistant ul {
+    margin: .25rem 0;
+    word-break: break-word;
+    line-height: 1.4;
+    font-size: 15px;
+    color: var(--ce-grey-80);
+}
+
+/* form elements */
+.mdh-assistant input,
+.mdh-assistant button {
+    min-height: var(--touch);
+    min-width: var(--touch);
+    border: 0;
+    border-radius: var(--border-radius);
+}
+
+.mdh-assistant input {
+    flex: 1;
+    min-height: calc(var(--touch) + 4px);
+    min-width: calc(var(--touch) + 4px);
+    padding: 5px 50px 5px 15px;
+    font-size: 1rem;
+    color: var(--ce-grey-80);
+}
+
+.mdh-assistant button {
+    background: transparent;
+    border: 2px solid transparent;
+    cursor: pointer;
+    color: var(--mdh-blue-55);
+    position: absolute;
+    right: 4px;
+}
+
+.mdh-assistant button:hover {
+    background: var(--mdh-blue-5);
+    border: 2px solid transparent;
+}
+
+.mdh-assistant .input-group {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    background: #fff;
+    border-top: var(--border);
+}
+
+/* other */
+.mdh-assistant input {
+    padding: 15px 35px 15px 15px;
+    box-sizing: border-box;
+    margin: 2px 3px;
+    width: calc(100% - 6px);
+}
+
+.mdh-assistant .ai-message,
+.mdh-assistant .user-message,
+.mdh-assistant #loading {
+    padding: 4px 8px 4px 20px;
+    position: relative;
+    border-bottom: 1px solid var(--ce-grey-5);
+}
+
+.mdh-assistant .user-message {
+    background: var(--ce-grey-3);
+    min-height: 27px;
+}
+
+.mdh-assistant .ai-message .svg-inline--fa,
+.mdh-assistant .user-message .svg-inline--fa,
+.mdh-assistant #loading .svg-inline--fa {
+    color: var(--ce-grey-40);
+    position: absolute;
```

Comment:
I would display:flex this with the message and not position it absolutely actually